plugins/background_fetch/background_fetch.cc

```diff
@@ -49,6 +49,10 @@ typedef std::unordered_map<std::string, bool> OutstandingRequests;
 class BgFetchState
 {
 public:
+  BgFetchState() {}
```

Comment:
I'm surprised your style plugin didn't complain about this. The style now is
```
BgFetchState() = default;
```
rather than empty braces. There's no difference in the complied code.

Reply:
Ok.